src/mixin/entity-mixin-lit.js

```diff
@@ -35,7 +35,7 @@ export const EntityMixinLit = superclass => class extends superclass {
 			this.href && this.token) {
 			this._getEntity();
 		}
-		return this.href && this.token;
+		return this.href && this.token ? super.shouldUpdate(changedProperties) : false;
```

Comment:
Actually... `localize-mixin.js` had to do a bunch of extra work to accomplish something similar, so I'm worried something is getting lost here. Like it's remembering the list of updated properties when it returns `false` so that later when it can call `super.shouldUpdate(changedProperties)` it can include all the ones that it skipped.

Reply:
Oh interesting, I can look into that

Reply:
It's possible that it's overkill... you could probably just remember if `super.shouldUpdate(changedProperties)` would have ever returned `true` (when you're returning `false` because `href`/`token` aren't defined yet) and then if so, return that value.

Reply:
Yeah interesting, I think it's overkill for the use cases I know about because the `localizeMixin` doesn't actually _use_ the `changedProperties`.  But there could be another mixin after it that does... could this be a controller mixins could use? 🤔